src/main/java/com/tieumy/exercise/OOP/Exercise01/Point.java

```diff
@@ -0,0 +1,23 @@
+package com.tieumy.exercise.OOP.Exercise01;
+
+public class Point {
+    private final double x;
+    private final double y;
+
+    public Point(final double x, final double y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public double getX() {
+        return x;
+    }
+
+    public double getY() {
+        return y;
+    }
+
+    public static double distanceTo(final Point a, final Point b) {
```

Comment:
Mình bỏ 1 tham số đi

```suggestion
    public double distanceTo(final Point point) {
```

Thay vào đó, dùng chính object hiện tại để so sánh. Tức là `b.getX()` sẽ thành `point.getX()` còn `a.getX()` đơn giản là `x` hoặc `getX()`

Reply:
DONE